tests/fixtures/migrations/await_precedence.php.expect

```diff
@@ -0,0 +1,30 @@
+<?hh // strict
+
+/**
+ * Copyright (c) 2016, Facebook, Inc.
```

Comment:
where's this copied from?

- it should be `*  Copyright (c) 2017-present, Facebook, Inc.`
- it should start with `/*`, not `/**` - `/**` marks a docblock, so with `/**` this is parsed as documentation for the function `foo`  below. It should just be a comment, not a docblock

Reply:
Hmm, I don't remember where I got this one from. Easy enough to change though.